v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
```

Comment:
> But then we need to think about the source and destination languages. If I call https://en.wikipedia.org/api/rest_v1/transform/html/language/..., then is en the source or the destination language? Obviously, because we already have the domain, we don't need two languages, but which of these do we not include?

Yeah, this is the "source" vs. "destination" question. For requesting existing content from a wiki in another language, I personally find the "source" variant more intuitive (request to enwiki to get translated enwiki content), but for a translator creating content for a specific target wiki, the other way around makes sense too. For pure transforms (where both input & output can presumably be arbitrary languages), I don't have any intuition at all. So agreed,

> It's not an easy answer. 

What do you think about using "html" as the source type, vs. "machine" or "engine"?